### ISSUE TITLE
EXR Driver missing on Windows

### ISSUE THREAD

**Issue by bertt** [MaxRev-Dev/gdal.netcore#139]
Hi, just upgraded my sample to 3.9.0, I noticed that GDAL driver 'EXR' is missing on Windows (150 gdal drivers), not on Linux/Mac (151 gdal drivers).

**Comment by MaxRev-Dev** [comment]
Yes, that's intended. OpenEXR is [not recognized ](https://github.com/MaxRev-Dev/gdal.netcore/actions/runs/9423867872/job/25963071496#step:6:1100) on Windows during configuration.

I have not found any solution to this. If you know how to enable it, you're welcome to contribute.

https://github.com/MaxRev-Dev/gdal.netcore/blob/4428f2cec8c363648b718bac3185d80775941816/win/partials.psm1#L253-L261
